client/src/components/Button/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC, ReactNode } from 'react';
+import styles from './styles.module.scss';
+
+interface ButtonProps {
+  type: 'button' | 'reset' | 'submit' | undefined;
+  children: ReactNode;
+  className: string;
+}
+
+const Button: FC<ButtonProps> = ({ type, children, className }) => {
+  return (
+    <div>
+      <button type={type} className={`${styles[className]}`}>
```

Comment:
would be better to have some prop like "variant"/"color"/etc, define it as a enum, and use special classname according to this prop provided. because now developer needs to additionally go to `styles.module.scss` file to take a look what classnames are available. and if you'll rename "searchBtn" class name, it should be renamed across the app, that makes your component less independent.
or another option is just to move `searchBtn` inside styles of component where `Button` is used